src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using Internal.Runtime.CompilerServices;
+
+namespace System.Numerics
+{
+    /* Note: The following patterns are used throughout the code here and are described here
+    *
+    * PATTERN:
+    *    if (typeof(T) == typeof(int)) { ... }
+    *    else if (typeof(T) == typeof(float)) { ... }
+    * EXPLANATION:
+    *    At runtime, each instantiation of Vector<T> will be type-specific, and each of these typeof blocks will be eliminated,
+    *    as typeof(T) is a (JIT) compile-time constant for each instantiation. This design was chosen to eliminate any overhead from
+    *    delegates and other patterns.
+    *
+    * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+    /// <summary>
+    /// A structure that represents a single Vector. The count of this Vector is fixed but CPU register dependent.
+    /// This struct only supports numerical types. This type is intended to be used as a building block for vectorizing
+    /// large algorithms. This type is immutable, individual elements cannot be modified.
+    /// </summary>
+    [Intrinsic]
+    public struct Vector<T> : IEquatable<Vector<T>>, IFormattable where T : struct
+    {
+        private ulong _00;
```

Comment:
The type contains 2x `ulong` now, rather than the explicit layout `Register` struct (which was a 16-byte, 66 field explicit layout struct).

Reply:
I had to add back `Register`. ARM64 was failing due to some disconnect between the HFA classification causing it to be `hasMultiregStructArgs` and `fgMorphMultiregStructArgs` not reporting the type as `TYP_STRUCT`.